periflow/sdk/resource/checkpoint.py

```diff
@@ -822,6 +822,7 @@ def convert(
 
         """
         try:
+            import accelerate  # type: ignore[import] # pylint: disable=import-outside-toplevel
```

Comment:
please move this statement to the top of the file, where all import statements are grouped

Reply:
It's because when moving it, the circular import error occurs in `periflow/sdk/resource/checkpoint.py`. For resolving, I think seperating `convert from this file need.

Reply:
it does not raise an error for me, can you share the error log?

Reply:
When using `pip install periflow-client` without mllib, then it raise an error. 

> Traceback (most recent call last):
  File "/Users/seungah/workspace/periflow-client/venv/bin/pf", line 3, in <module>
    from periflow.cli import app
  File "/Users/seungah/workspace/periflow-client/periflow/__init__.py", line 12, in <module>
    from periflow.sdk.resource.checkpoint import Checkpoint
  File "/Users/seungah/workspace/periflow-client/periflow/sdk/resource/checkpoint.py", line 13, in <module>
    import torch 
    ^^^^^^^^^^^^
ModuleNotFoundError: No module named 'torch'